src/js/entities-service/form-responses.js

```diff
@@ -9,11 +9,17 @@ const Entity = BaseEntity.extend({
     'formResponses:model': 'getModel',
     'formResponses:collection': 'getCollection',
     'fetch:formResponses:submission': 'fetchSubmission',
+    'fetch:formResponses:latestSubmission': 'fetchLatestSubmission',
   },
   fetchSubmission(responseId) {
     if (!responseId) return [{}];
     return $.ajax(`/api/form-responses/${ responseId }/response`);
   },
+  fetchLatestSubmission(patientId, formId) {
+    if (!patientId || !formId) return [{}];
```

Comment:
I'm not sure about this guard.  I know there's one above, but that's almost certainly because that request is called without the responseId.  It might be impossible to test this guard, and if so we should remove it.

Reply:
Okay makes sense, I'll have that removed in the next commit.